File: Web/src/app/Entities/Pagos.ts
export class Pagos {

  pagoId : number;
  hipoId: number;
  pagoFecha : Date;
  pagoMonto: number;
  pagoMontoAnticipado: number;
  pagoMesAnticipado: number;

  constructor() {
    this.pagoId = 0;
    this.hipoId = 0;
    this.pagoMonto = 0;
    this.pagoMontoAnticipado = 0;
    this.pagoFecha = new Date();
    this.pagoMesAnticipado = 0;
  }
}

File: Api/HipotecaSimple/Data/Entities/Pagos.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace HipotecaSimple.Data.Entities
{
    public class Pagos
    {
        [Key]
        public int PagoId { get; set; }
        public int HipoId { get; set; }
        public DateTime PagoFecha { get; set; }
        public decimal PagoMonto { get; set; }
        public decimal PagoMontoAnticipado { get; set; }
        public int PagoMesAnticipado { get; set; }
    }

}

